src/api/Themer.ts

```diff
@@ -49,11 +49,30 @@ export function applyTheme() {
         setAMOLEDThemeEnabledBypass(true);
         logger.info("Theme applied: ", currentTheme.name);
 
+        Dialog.show({
+            title: "Restart for theme to apply",
+            body: "Restart the app for the theme to apply correctly.",
+            confirmText: "Restart",
+            isDismissable: false,
+            onConfirm: ReactNative.NativeModules.BundleUpdaterManager.reload
+        });
+
     } catch (e) {
+        Dialog.show({
+            title: "Failed to apply theme",
+            body: `${currentTheme.name} failed to apply. Please report this issue. Theme will be disabled on restart.`,
+            isDismissable: false,
+            cancelText: "Do not restart",
+            confirmText: "Restart",
+            onConfirm: ReactNative.NativeModules.BundleUpdaterManager.reload
+        });
+
         logger.error("Failed to apply theme: ", e);
+        window.Aliucord.settings.set("theme", "");
         
-        unpatchInput();
-        unpatchNav();
+        // Unpatch components
+        unpatchInput;
+        unpatchNav;
```

Comment:
? what is this supposed to do

Reply:
hermes crashes the app if i put (), its the unpatch funcs